.github/workflows/playwright.yml

```diff
@@ -40,12 +40,12 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
 
-      - name: Attempt to wait for deploy to environment (10 minutes sleep)
+      - name: Attempt to wait for deploy to environment (15 minutes sleep)
```

Comment:
Needed to increase, since our e2e-tests was starting before our latest version of designer was deployed to dev.